setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = markdown-to-confluence
+name = md2conf
```

Comment:
The name of the package in PyPI is indeed `markdown-to-confluence`. Unfortunately, when I was about to publish *md2conf* to PyPI, the name `md2conf` was already taken.